src/erlcloud_ddb2.erl

```diff
@@ -3677,7 +3678,6 @@ transact_write_items(TransactItems, Opts, Config) ->
     case out(Return,
              fun(Json, UOpts) -> undynamize_record(transact_write_items_record(), Json, UOpts) end, DdbOpts,
              #ddb2_transact_write_items.attributes, {ok, []}) of
-        {simple, Record} -> {ok, Record};
```

Comment:
is this right?

Reply:
It's further explained in commit 078fae2's message: 

> `out/5` never returns `{simple, _}` which is why it was removed from the `case` in `transact_write_items/3`

That function is even `-spec(_)`ed to return `-> ok_return(term())`.

Reply:
Perfect, thanks. Sorry, was looking at the unified view.